app/gzac/src/main/java/com/ritense/gzac/listener/ApplicationReadyEventListener.kt

```diff
@@ -22,6 +25,20 @@ class ApplicationReadyEventListener(
     fun handle() {
         connectorService.getConnectorTypes().forEach {
             try {
+                if (it.name == "OpenZaak") {
+                    connectorService.createConnectorInstance(
+                        typeId = it.id.id,
+                        name = it.name.replace("\\s".toRegex(), "") + "Instance",
+                        connectorProperties = OpenZaakProperties(
+                            OpenZaakConfig(
+                                "http://localhost:8001",
+                                "valtimo_client",
+                                "e09b8bc5-5831-4618-ab28-41411304309d",
+                                Rsin("051845623")
```

Comment:
Hardcoded configuratie?
Dat lijkt me niet de bedoeling

Reply:
Ja, het is een library project en daar hoort dit soort code niet in te staat, nee.
Het idee is: deze /app/gzac -module is niet bruikbaar zonder deze configuratie. En aangezien we dit ook hebben gedaan voor ObjectsApi hieronder.